db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1972,9 +1969,6 @@ Status DBImpl::RunManualCompaction(
                manual.begin, manual.end, &manual.manual_end, &manual_conflict,
                max_file_num_to_ignore, trim_ts)) == nullptr &&
           manual_conflict))) {
-      // exclusive manual compactions should not see a conflict during
-      // CompactRange
-      assert(!exclusive || !manual_conflict);
```

Comment:
[Note] Unfortunately, exclusive manual compaction can have *manual_conflict == true since it can conflict with file ingestion after we add  file ingestion to conflict check ADN remove WaitForIngestFile(); I think it's fine to remove the assertion as it's more of a defensive assertion.